CMakeLists.txt

```
@@ -93,9 +96,15 @@ target_link_libraries(rmm INTERFACE CCCL::CCCL)
 target_link_libraries(rmm INTERFACE fmt::fmt-header-only)
 target_link_libraries(rmm INTERFACE spdlog::spdlog_header_only)
 target_link_libraries(rmm INTERFACE dl)
+target_link_libraries(rmm INTERFACE $<BUILD_LOCAL_INTERFACE:nvtx3-cpp>)
```

I checked with @KyleFromNVIDIA. This might fix the build failure in rmm Python:
```suggestion
target_link_libraries(rmm INTERFACE nvtx3-cpp)
```